libs/live-nft-react/src/index.ts

```diff
@@ -5,4 +5,5 @@ export * from "./hooks/useNftFloorPrice";
 export * from "./hooks/useRefreshMetadata";
 export * from "./hooks/useCheckSpamScore";
 export * from "./hooks/useFetchOrdinals";
+export * from "./hooks/helpers/utxoCompliant";
```

Comment:
it's not well structured here, 

`helpers` should be outside `hooks` imo but it's ok :)

Reply:
moved to common/currencies/helper finally